examples/hello_world.py

```diff
@@ -9,4 +9,6 @@
 from bcc import BPF
 
 # This may not work for 4.17 on x64, you need replace kprobe__sys_clone with kprobe____x64_sys_clone
-BPF(text='int kprobe__sys_clone(void *ctx) { bpf_trace_printk("Hello, World!\\n"); return 0; }').trace_print()
+BPF(
+    text='int kprobe__sys_clone(void *ctx) { bpf_trace_printk("Hello, World!\\n"); return 0; }'
+).trace_print()
```

Comment:
This intentionally to be one-liner. Could you restore to one line and add necessary comments so that flake8 will ignore this one?